src/transformers/models/data2vec/configuration_data2vec_audio.py

```diff
@@ -71,13 +71,13 @@ class Data2VecAudioConfig(PretrainedConfig):
         feat_extract_activation (`str, `optional`, defaults to `"gelu"`):
             The non-linear activation function (function or string) in the 1D convolutional layers of the feature
             extractor. If string, `"gelu"`, `"relu"`, `"selu"` and `"gelu_new"` are supported.
-        conv_dim (`Tuple[int]`, *optional*, defaults to `(512, 512, 512, 512, 512, 512, 512)`):
+        conv_dim (`Union[Tuple[int], List[int]]`, *optional*, defaults to `(512, 512, 512, 512, 512, 512, 512)`):
```

Comment:
We usually use English here:
```suggestion
        conv_dim (`Tuple[int]` or `List[int]`, *optional*, defaults to `(512, 512, 512, 512, 512, 512, 512)`):
```
(also not sure we actually need to refine it that much). This applies here and in all other config files changed in this PR.